### DISCUSSION TITLE
Unable to register Sendgrid account

### DISCUSSION THREAD

**Discussion by bhav380-2** [cn10xdev/Event-Management#11] [Q&A]
When i am trying to register account in sendGrid (https://sendgrid.com/en-us), i am getting this error message "!We are unable to log you in at this time, please check your email for further information."

And in email I have received email from SendGrid team which reads:-
"We appreciate your interest in Twilio SendGrid and your efforts in completing our account creation process. After a thorough review, we regret to inform you that we are unable to proceed with activating your account (unified_acct_**id) at this time.

Ensuring the security and integrity of our platform is our top priority, and our vetting process is designed to detect potential risks. While we understand the importance of transparency, we are not able to provide the specifics of our vetting process.

We want to emphasize that our decision is based on stringent security measures and our commitment to the safety of all our users."

I tried registering using multiple gmail accounts but getting this same message in email.

![Screenshot from 2024-09-22 18-16-52](https://github.com/user-attachments/assets/0aaeb1e5-cb2b-4056-95de-2e38995e31fd)

![Screenshot from 2024-09-22 18-18-30](https://github.com/user-attachments/assets/a070774d-c86d-4e73-a6f4-df53403b9d68).

**Comment by bhardwajdisha** [comment]
It seems that Sendgrid is not allowing new users to create an account at the moment. For now, you can comment out that part of the code and continue working on the rest of the project. Alternatively, you can raise an issue regarding the Sendgrid account issue and begin working on switching the mailing system to another library. Once done, you can raise a PR for the changes.

**Reply by bhav380-2** [reply]
Sure

**Comment by bhardwajdisha** [comment]
Another option is that I can handle this for you if you'd prefer, but it seems like a great issue for you to work on your own.

**Reply by bhav380-2** [reply]
Yes sure , I will take care of it.
